Userland/Libraries/LibCompress/BrotliDictionary.cpp

```diff
@@ -0,0 +1,7908 @@
+/*
+ * Copyright (c) 2022, Michiel Visser <opensource@webmichiel.nl>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <AK/Types.h>
+#include <LibCompress/BrotliDictionary.h>
+
+namespace Compress {
+
+static u8 dictionary_data[122784] {
+    0x74, 0x69, 0x6d, 0x65, 0x64, 0x6f, 0x77, 0x6e, 0x6c, 0x69, 0x66, 0x65, 0x6c, 0x65, 0x66, 0x74,
```

Comment:
Can this be stored in a binary format somehow? It feels pretty wasteful to have it in source code.